native/swift/Tests/wordpress-api/UUIDTests.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import XCTest
+import WordPressAPI
+
+class UUIDTests: XCTestCase {
+
+    func testConvertToUUID() {
+        let uuid = WpUuid().uuidString()
+        XCTAssertNotNil(UUID(uuidString: uuid), "WpUuid \(uuid) is not a Foundation.UUID")
+    }
+
```

Comment:
A couple of other test cases that may be valuable:

1. Ensure that rust-generated UUIDs are valid according to WP
2. Ensure that Swift-generated UUIDs are made valid by Rust

```suggestion
    // Copied from WordPress Core: 
    // https://github.com/WordPress/wordpress-develop/blob/471a619/src/wp-includes/functions.php#L7912
    let wpDefaultRegex = #/^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/#
    let wpv4Regex      = #/^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/#

    func testGeneratedUUIDIsValid() throws {
        XCTAssertNotNil(try wpDefaultRegex.wholeMatch(in: WpUuid().uuidString()))
        XCTAssertNotNil(try wpv4Regex.wholeMatch(in: WpUuid().uuidString()))
    }

    func testParsedAllCapsUUIDIsValid() throws {
        XCTAssertNotNil(try wpDefaultRegex.wholeMatch(in: try WpUuid.parse(input: UUID().uuidString).uuidString()))
        XCTAssertNotNil(try wpv4Regex.wholeMatch(in: try WpUuid.parse(input: UUID().uuidString).uuidString()))
    }

```